### DISCUSSION TITLE
Ability to backdate a post

### DISCUSSION THREAD

**Discussion by j2em6h** [forem/forem#20052] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Unable to change publish date. I personally wish to back date a post, but there is no way to set the publish date (or time) for a post.

**Describe the solution you'd like**
Add a custom variable for `publish_date`

**Describe alternatives you've considered**
Time travel?

**Additional context**
In lieu of being able to delete/edit comments on an old post I have duplicated and republished it as a new post, but the date does not/cannot reflect the origin publish date. 

Semi related to #3274 and #1363

**Comment by jessleenyc** [comment]
This makes sense - for us it would be figuring out where best to put this functionality (though I think the answer is on the /manage page). Question though -- if a post is backdated, would we still surface it as new content?

**Comment by j2em6h** [comment]
I was imagining it in the post editor with the other custom variables. I didn't even know /manage existed until just now; that seems like an ok spot for it too.

Personally, I think _not_ surfacing a back dated post as "new" makes sense, and is probably easier to implement.

**Comment by maestromac** [comment]
We actually already have this functionality in place but only for RSS articles. You'll notice the `Originally published at Medium on Jun 05, 2019` in [this article](https://dev.to/vaidehijoshi/weeding-out-distributed-system-bugs-9c8). That field will be replaced with `Updated at` if the article is ever updated.

**Comment by jessleenyc** [comment]
@maestromac I think we're talking about providing this for all articles.

**Comment by cdvillard** [comment]
I'm running into a similar issue when it comes to simply posting to The Practical Dev using the API. In my case, I'm attempting to use Netlify CMS to write a blog post and then send a POST request with the contents of that post after a build step. Within the Netlify CMS, I have it configured to use a date property to capture the publish date, but as there's no available field in the YAML of a DEV post, I fear it wouldn't take.

**Comment by kkgthb** [comment]
I'd also like to be able to backdate a post.  I realized that a series needed a "table of contents."  On my blog, I used the "date" front-matter property to bump it up to be first in my series.

But I can't figure out a way to change its date in Dev so that it shows up as the 1st post in the series auto-generator widget.

I don't want to delete and redo all my posts in DEV because I like my "originally published at" stuff from my feed and don't want to lose it.

**Comment by cmgorton** [comment]
Since this issue has gone stale I am going to close it. If anyone would still like this feature let me know and I can convert this issue to a [discussion](https://github.com/forem/forem/discussions)

**Comment by astrojuanlu** [comment]
> f anyone would still like this feature let me know and I can convert this issue to a [discussion](https://github.com/forem/forem/discussions)

Yes please 🙏🏽

**Comment by michael-tharrington** [comment]
@astrojuanlu I got you!

**Comment by calumk** [comment]
This feature would still be really helpful.....

**Comment by harmony7** [comment]
I would find this useful too.

**Comment by safeercm** [comment]
This is a useful feature, upvoting

**Comment by iandouglas** [comment]
Would also like to have this. We have dozens of posts from our engineering blog that we'd love to cross-post to dev.to but only if we can back-date them to the original publication date.
